mobile/salto/src/utils.ts

```diff
@@ -37,6 +38,7 @@ export function increaseNodeBalance(nodes: NodeData[], balanceChange: BalanceCha
             unit: balanceChange.unit,
             amount: balanceChange.amount, 
           };
+          priceProviderAddAssets([balanceChange.unit]);
```

Comment:
Not inside the front-end `increaseNodeBalance` function.
`priceProviderAddAssets` is currently a command. A methods that does front-end updatate (visualization) has no reason to call a command. Only user action should be doing that.

So I think there is a bad design here and that the `priceProviderAddAsseets` should not exist.
It should be handled during the `receiveWad` section, where the backend has all the data it needs to process the fact that a new asset is being stored in the wallet.
This way we avoid a back and forth